package.json

```diff
@@ -17,53 +17,37 @@
   "license": "MIT",
   "homepage": "https://github.com/stimulus-components/stimulus-timeago",
   "private": false,
-  "esnext": "dist-src/index.js",
-  "main": "dist-node/index.js",
-  "module": "dist-web/index.js",
-  "@pika/pack": {
-    "pipeline": [
-      [
-        "@pika/plugin-standard-pkg"
-      ],
-      [
-        "@pika/plugin-build-web"
-      ],
-      [
-        "@pika/plugin-build-node"
-      ]
-    ]
-  },
+  "main": "dist/stimulus-timeago.js",
+  "module": "dist/stimulus-timeago.js",
   "scripts": {
     "format": "prettier-standard  '**/*.{js,css,html}' --format",
     "lint": "prettier-standard  '**/*.{js,css,html}' --lint",
-    "dev": "snowpack dev",
-    "prod": "snowpack build",
-    "build": "pika build",
+    "dev": "vite",
+    "build": "tsc --noEmit && vite build",
     "test": "BABEL_ENV=test jest",
-    "pika:publish": "pika publish --no-tests --no-2fa",
-    "version": "yarn build"
+    "publish": "np --no-2fa"
   },
   "dependencies": {
-    "date-fns": "^2.17.0",
-    "stimulus": "^2.0.0"
+    "date-fns": "^2.21.1"
   },
   "devDependencies": {
-    "@babel/core": "7.12.16",
-    "@babel/plugin-syntax-class-properties": "7.12.13",
+    "@babel/core": "^7.14.0",
     "@babel/polyfill": "^7.12.1",
-    "@babel/preset-env": "^7.12.16",
-    "@pika/pack": "0.5.0",
-    "@pika/plugin-build-node": "0.9.2",
-    "@pika/plugin-build-web": "0.9.2",
-    "@pika/plugin-standard-pkg": "0.9.2",
-    "@snowpack/plugin-babel": "2.1.6",
-    "@testing-library/dom": "^7.29.6",
-    "@types/jest": "^26.0.20",
+    "@babel/preset-env": "^7.14.1",
+    "@babel/preset-typescript": "^7.13.0",
+    "@testing-library/dom": "^7.30.4",
+    "@types/jest": "^26.0.23",
     "babel-jest": "^26.6.3",
     "jest": "^26.6.3",
     "mutationobserver-shim": "^0.3.7",
+    "np": "^7.5.0",
     "prettier-standard": "16.4.1",
-    "snowpack": "2.14.0"
+    "stimulus": "^2.0.0",
+    "typescript": "^4.2.4",
+    "vite": "^2.2.4"
+  },
+  "peerDependencies": {
+    "stimulus": "^2.0.0"
```

Comment:
@guillaumebriday if you want force version node you can add to package.json this part of code:

`  "engines": {
    "node": ">=14.6.1",
  },
  `